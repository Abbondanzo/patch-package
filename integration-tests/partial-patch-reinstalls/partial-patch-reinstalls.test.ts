import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "partial-patch-reinstalls",
  shouldProduceSnapshots: true,
})
