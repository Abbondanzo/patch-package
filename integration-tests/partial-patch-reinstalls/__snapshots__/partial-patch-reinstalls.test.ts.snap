// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test partial-patch-reinstalls: left-pad should contain patch-package 1`] = `
"SNAPSHOT: left-pad should contain patch-package
    // devide \`len\` by 2, ditch the patch-package
END SNAPSHOT"
`;
